### PR TITLE
Menu Commands fixed in search

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
References issue #3382

### CODE REVIEW

File: src/js/actions/search/commands.js [hunk 1]
```diff
@@ -123,13 +123,11 @@ define(function (require, exports) {
     var _menuCommandSearchOptions = function () {
         var menuStore = this.flux.store("menu"),
             menu = menuStore.getApplicationMenu(),
-            menuMap = menu.rootMap,
             roots = menu.roots.reverse();
 
         var menuCommands = [];
         roots.forEach(function (root) {
-            var currRoot = menuMap.get(root.id),
```
Comment (iwehrman) [inline src/js/actions/search/commands.js]: Do you know what the purpose of this line was previously? It looks superfluous to me, but I just want to confirm that you agree. (The same functionality is still available via `MenuBar.prototype.byID`, btw.)

Comment (mcilroyc) [inline src/js/actions/search/commands.js]: Looks to me like it was just a misunderstanding during the original implementation.... like maybe it seemed that it was necessary to get the root from the map, event though it wasn't.

Comment (DivyaPrabhakar) [inline src/js/actions/search/commands.js]: @iwehrman @mcilroyc I agree about it being a misunderstanding -- it is odd that it was working before and I had the same question of why this was here in the first place. But yes both of your observations are correct, it was just a small mistake in what reference was being inserted into the array.

**Comment by mcilroyc** [comment]
Seems legit.  merging.

**Pull request merged**